{
  "checkpoints": [
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00007\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"7\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00007\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329372.3384254,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00007_7_2022-10-09_17-29-31\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00007_7_2022-10-09_17-29-31/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224705, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7fc5d5fadf10>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224705, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7fc5d5fadf10>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00005\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"5\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00005\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329365.4025335,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00005_5_2022-10-09_17-29-24\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00005_5_2022-10-09_17-29-24/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224696, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7fc8b1a1f100>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224696, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7fc8b1a1f100>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00006\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"6\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00006\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329371.76612,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00006_6_2022-10-09_17-29-25\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00006_6_2022-10-09_17-29-25/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224703, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f1ea11a05e0>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224703, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f1ea11a05e0>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00003\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"3\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00003\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329358.194602,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00003_3_2022-10-09_17-29-17\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00003_3_2022-10-09_17-29-17/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224702, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f261a01ed90>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224702, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f261a01ed90>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00000\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"0\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {},\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329345.266712,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00000_0_2022-10-09_17-29-05\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00000_0_2022-10-09_17-29-05/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224693, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f3ace5fce20>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224693, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f3ace5fce20>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00001\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"1\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00001\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329351.430442,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00001_1_2022-10-09_17-29-11\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00001_1_2022-10-09_17-29-11/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224713, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f265694ac40>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224713, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f265694ac40>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00009\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"9\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00009\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329378.9241364,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00009_9_2022-10-09_17-29-38\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00009_9_2022-10-09_17-29-38/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224697, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f32060225e0>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224697, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f32060225e0>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00002\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"2\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00002\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329357.5971172,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00002_2_2022-10-09_17-29-17\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00002_2_2022-10-09_17-29-17/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224712, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7fce5541b670>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224712, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7fce5541b670>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00004\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"4\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00004\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329364.7848384,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00004_4_2022-10-09_17-29-18\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00004_4_2022-10-09_17-29-18/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224716, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f771baab970>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224716, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f771baab970>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}",
    "{\n  \"stub\": false,\n  \"trainable_name\": \"training\",\n  \"trial_id\": \"1bf44_00008\",\n  \"config\": {\n    \"a_t\": 0.95,\n    \"a_t_max\": 1.0,\n    \"h_t\": 0.8,\n    \"h_t_max\": 1.0,\n    \"kappa\": 1.05,\n    \"tau_b\": 0.3,\n    \"tau_n\": 0.1,\n    \"max_age\": 10,\n    \"eps_b\": 0.1,\n    \"eps_n\": 0.001,\n    \"beta\": 0.7,\n    \"n_context\": 4,\n    \"phi\": 0.7,\n    \"neighborhood\": true,\n    \"mem_batch_size\": 500,\n    \"amount_real_data\": 0,\n    \"td_modulate\": false,\n    \"log_freq\": 1,\n    \"load_latest\": false,\n    \"policy\": \"DDPG\",\n    \"env\": \"Reacher-v2\",\n    \"seed\": 42,\n    \"start_timesteps\": 15000.0,\n    \"eval_freq\": 2500,\n    \"max_timesteps\": 100000.0,\n    \"expl_noise\": 0.1,\n    \"batch_size\": 64,\n    \"discount\": 0.99,\n    \"tau\": 0.005,\n    \"policy_noise\": 0.2,\n    \"noise_clip\": 0.5,\n    \"policy_freq\": 2,\n    \"save_model\": false,\n    \"load_model\": \"\",\n    \"replay_memory\": \"original\",\n    \"logging\": true,\n    \"run_name\": \"221009_17-29\",\n    \"batch_name\": \"Baseline\",\n    \"log_path\": \"/data/8immisch/gwr-ddpg/src\",\n    \"save_network\": false,\n    \"ray_tune\": true,\n    \"resume_ray\": false,\n    \"num_ray_runs\": 10,\n    \"run_schedule\": null,\n    \"scheduler\": null,\n    \"search_alg\": null,\n    \"ray_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/\"\n  },\n  \"local_dir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline\",\n  \"evaluated_params\": {},\n  \"experiment_tag\": \"8\",\n  \"location\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"8005953a000000000000008c0e7261792e74756e652e747269616c948c084c6f636174696f6e9493942981947d94288c08686f73746e616d65944e8c03706964944e75622e\"\n  },\n  \"placement_group_factory\": \"800595b6000000000000008c1f7261792e74756e652e7574696c732e706c6163656d656e745f67726f757073948c15506c6163656d656e7447726f7570466163746f72799493942981947d94288c155f686561645f62756e646c655f69735f656d70747994898c085f62756e646c6573945d947d94288c03435055944740180000000000008c0347505594473ff000000000000075618c095f7374726174656779948c045041434b948c055f6172677394298c075f6b7761726773947d9475622e\",\n  \"stopping_criterion\": {\n    \"total_t\": 100000.0\n  },\n  \"log_to_file\": [\n    null,\n    null\n  ],\n  \"max_failures\": 0,\n  \"_last_result\": {\n    \"trial_id\": \"1bf44_00008\"\n  },\n  \"_default_result_or_future\": null,\n  \"last_update_time\": -Infinity,\n  \"metric_analysis\": {},\n  \"n_steps\": [\n    5,\n    10\n  ],\n  \"metric_n_steps\": {},\n  \"export_formats\": [],\n  \"status\": \"ERROR\",\n  \"start_time\": 1665329378.5061612,\n  \"logdir\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00008_8_2022-10-09_17-29-32\",\n  \"runner\": null,\n  \"last_debug\": 0,\n  \"error_file\": \"/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/training_Reacher-v2_1bf44_00008_8_2022-10-09_17-29-32/error.txt\",\n  \"error_msg\": \"Traceback (most recent call last):\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trial_runner.py\\\", line 924, in _process_trial\\n    results = self.trial_executor.fetch_result(trial)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/ray_trial_executor.py\\\", line 787, in fetch_result\\n    result = ray.get(trial_future[0], timeout=DEFAULT_GET_TIMEOUT)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/_private/client_mode_hook.py\\\", line 105, in wrapper\\n    return func(*args, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/worker.py\\\", line 1713, in get\\n    raise value.as_instanceof_cause()\\nray.exceptions.RayTaskError(TuneError): \\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224692, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f6dea8fd2e0>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 255, in train_buffered\\n    result = self.train()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/trainable.py\\\", line 314, in train\\n    result = self.step()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 381, in step\\n    self._report_thread_runner_error(block=True)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 531, in _report_thread_runner_error\\n    raise TuneError(\\nray.tune.error.TuneError: Trial raised an exception. Traceback:\\n\\u001b[36mray::ImplicitFunc.train_buffered()\\u001b[39m (pid=3224692, ip=134.100.10.165, repr=<types.ImplicitFunc object at 0x7f6dea8fd2e0>)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 262, in run\\n    self._entrypoint()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 330, in entrypoint\\n    return self._trainable_func(self.config, self._status_reporter,\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/ray/tune/function_runner.py\\\", line 597, in _trainable_func\\n    output = fn()\\n  File \\\"/data/8immisch/gwr-ddpg/src/train.py\\\", line 53, in training\\n    env = gym.make(args[\\\"env\\\"])\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 235, in make\\n    return registry.make(id, **kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 129, in make\\n    env = spec.make(**kwargs)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 89, in make\\n    cls = load(self.entry_point)\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/registration.py\\\", line 27, in load\\n    mod = importlib.import_module(mod_name)\\n  File \\\"/usr/lib/python3.8/importlib/__init__.py\\\", line 127, in import_module\\n    return _bootstrap._gcd_import(name[level:], package, level)\\n  File \\\"<frozen importlib._bootstrap>\\\", line 1014, in _gcd_import\\n  File \\\"<frozen importlib._bootstrap>\\\", line 991, in _find_and_load\\n  File \\\"<frozen importlib._bootstrap>\\\", line 975, in _find_and_load_unlocked\\n  File \\\"<frozen importlib._bootstrap>\\\", line 671, in _load_unlocked\\n  File \\\"<frozen importlib._bootstrap_external>\\\", line 848, in exec_module\\n  File \\\"<frozen importlib._bootstrap>\\\", line 219, in _call_with_frames_removed\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/__init__.py\\\", line 1, in <module>\\n    from gym.envs.mujoco.mujoco_env import MujocoEnv\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/gym/envs/mujoco/mujoco_env.py\\\", line 12, in <module>\\n    import mujoco_py\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/__init__.py\\\", line 3, in <module>\\n    from mujoco_py.builder import cymj, ignore_mujoco_warnings, functions, MujocoException\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/builder.py\\\", line 505, in <module>\\n    mujoco_path, key_path = discover_mujoco()\\n  File \\\"/data/8immisch/gwr-ddpg/.venv/lib/python3.8/site-packages/mujoco_py/utils.py\\\", line 93, in discover_mujoco\\n    raise Exception(message)\\nException: \\nYou appear to be missing MuJoCo.  We expected to find the file here: /export/home/8immisch/.mujoco/mujoco200\\n\\nThis package only provides python bindings, the library must be installed separately.\\n\\nPlease follow the instructions on the README to install MuJoCo\\n\\n    https://github.com/openai/mujoco-py#install-mujoco\\n\\nWhich can be downloaded from the website\\n\\n    https://www.roboti.us/index.html\\n\",\n  \"trial_name_creator\": null,\n  \"custom_trial_name\": null,\n  \"custom_dirname\": null,\n  \"saving_to\": null,\n  \"remote_checkpoint_dir_prefix\": null,\n  \"sync_function_tpl\": null,\n  \"checkpoint_freq\": 0,\n  \"checkpoint_at_end\": false,\n  \"keep_checkpoints_num\": null,\n  \"checkpoint_score_attr\": \"training_iteration\",\n  \"sync_on_checkpoint\": true,\n  \"checkpoint_manager\": {\n    \"_type\": \"CLOUDPICKLE_FALLBACK\",\n    \"value\": \"80059566010000000000008c1b7261792e74756e652e636865636b706f696e745f6d616e61676572948c11436865636b706f696e744d616e616765729493942981947d94288c146b6565705f636865636b706f696e74735f6e756d94477ff00000000000008c165f636865636b706f696e745f73636f72655f6465736394898c165f636865636b706f696e745f73636f72655f61747472948c12747261696e696e675f697465726174696f6e948c1c6e65776573745f70657273697374656e745f636865636b706f696e749468008c0a436865636b706f696e749493942981947d94288c0773746f72616765948c0a70657273697374656e74948c0576616c7565944e8c06726573756c74947d9475628c195f6e65776573745f6d656d6f72795f636865636b706f696e7494680b2981947d9428680e8c066d656d6f72799468104e68117d9475628c115f626573745f636865636b706f696e7473945d948c0b5f6d656d62657273686970948f9475622e\"\n  },\n  \"restore_path\": null,\n  \"restoring_from\": null,\n  \"num_failures\": 1,\n  \"has_new_resources\": false,\n  \"results\": \"80054e2e\",\n  \"best_result\": \"80054e2e\",\n  \"param_config\": \"80054e2e\",\n  \"extra_arg\": \"80054e2e\",\n  \"_state_json\": null,\n  \"_state_valid\": false\n}"
  ],
  "runner_data": {
    "_max_pending_trials": 70,
    "_metric": null,
    "_total_time": 0,
    "_iteration": 31,
    "_has_errored": true,
    "_fail_fast": false,
    "_server_port": null,
    "_cached_trial_decisions": {},
    "_queued_trial_decisions": {},
    "_updated_queue": true,
    "_result_wait_time": 1,
    "_should_stop_experiment": false,
    "_local_checkpoint_dir": "/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline",
    "_remote_checkpoint_dir": null,
    "_stopper": {
      "_type": "CLOUDPICKLE_FALLBACK",
      "value": "80059527000000000000008c107261792e74756e652e73746f70706572948c0b4e6f6f7053746f707065729493942981942e"
    },
    "_resumed": false,
    "_start_time": 1665329345.0729275,
    "_last_checkpoint_time": -Infinity,
    "_session_str": "2022-10-09_17-29-05",
    "checkpoint_file": "/data/8immisch/gwr-ddpg/src/results/ray_results/Reacher-v2/Baseline/experiment_state-2022-10-09_17-29-05.json",
    "_checkpoint_period": "auto",
    "launch_web_server": false
  },
  "stats": {
    "start_time": 1665329345.0729275,
    "timestamp": 1665329378.4984415
  }
}